lib/Models/GlobeOrMap.js

```diff
@@ -91,4 +91,14 @@ GlobeOrMap.prototype._createEntityFromImageryLayerFeature = function(feature) {
     return entity;
 };
 
+//set text attribution
```

Comment:
Use jsdoc style comments to document methods and properties.  The `DocBlockr` Sublime Text plugin helps you write them.